hibernate/src/main/java/com/example/homework6/EmailDeliver.java

```diff
@@ -0,0 +1,80 @@
+package com.example.homework6;
+
+import com.example.homework6.controllers.EmailController;
+import com.example.homework6.repos.EmailRepo;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Bean;
+import org.springframework.mail.MailException;
+import org.springframework.mail.SimpleMailMessage;
+import org.springframework.mail.javamail.JavaMailSender;
+import org.springframework.mail.javamail.JavaMailSenderImpl;
+import org.springframework.stereotype.Component;
+
+import java.util.Date;
+import java.util.Properties;
+
+@Component
+public class EmailDeliver extends Thread{
```

Comment:
It is much wiser to use _org.springframework.scheduling.concurrent.ThreadPoolTaskExecutor_.
In order not to deal with threads.